src/components/Navigation/Pages/PageBrowse/PageBrowse.js

```diff
@@ -63,15 +86,47 @@ class PageBrowse extends SystemManagedList {
         onFilter={this.onFilter}
         onSort={this.onSort}
         hasComponents={!this.hasComponents()}
+        showSourceFilter={false}
+        customLicenses={licenses.filter(license => license.value !== 'absence' && license.value !== 'presence')}
       />
     )
   }
 
   onProviderSelection(event) {
     const target = event.target
     const activeProvider = target.name
-    this.setState({ ...this.state, activeProvider })
-    this.props.dispatch(uiBrowseGet(this.props.token, { type: activeProvider }))
+    this.setState({ ...this.state, activeProvider, activeFilters: null, activeSort: null })
+    this.props.dispatch(
+      uiBrowseGet(this.props.token, { type: activeProvider, maxLicensedScore: 70, maxDescribedScore: 70 })
+    )
+  }
+
+  async updateData(continuationToken) {
+    const { activeFilters, activeSort, activeProvider } = this.state
+    const query = { type: activeProvider, maxLicensedScore: 70, maxDescribedScore: 70 }
```

Comment:
why set `maxDescribedScore` and `maxLicensedScore` to 70?

I think we should only be setting filters if they are picked by the user

Reply:
The initial request for the Browse page was to show only definitions with a score below 70

Reply:
i see.

Reply:
I just had a quick chat with @fossygirl on this.
Let's remove the <70 filter from the query by default and add a new 'curate' dropdown to turn on these filters

Here's a mockup of the experience

![image](https://user-images.githubusercontent.com/5168796/52440692-b2ef3d00-2ad3-11e9-8fe3-e51aa7a28e5d.png)

When you check 'licensed' filter by maxLicensedScore: 70
When you check 'described' filter by maxDescribedScore: 70

Reply:
Got it, i will implement this, thank you!